modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -510,15 +510,51 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
     using ParamType = std::pair<InferenceEngine::TensorDesc,
                                 InferenceEngine::ParamMap>;
 
-    ParamType* blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
-    if (blob_params == nullptr) {
+    ParamType* frame_blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
+    if (frame_blob_params == nullptr) {
         GAPI_Assert(false && "Incorrect type of blobParams: "
                               "expected std::pair<InferenceEngine::TensorDesc,"
                                                  "InferenceEngine::ParamMap>");
     }
 
-    return ctx.uu.rctx->CreateBlob(blob_params->first,
-                                   blob_params->second);
+    if (ctx.uu.params.device_id.find("GPU") != std::string::npos) {
+        GAPI_LOG_DEBUG(nullptr, "Extract remote blob for GPU DX11 device_id: " <<
+                                ctx.uu.params.device_id);
+        // despite of layout, blob dimensions always follow in N,C,H,W order
```

Comment:
Will only 4D blob be supported?

Reply:
it depends from openvino  and code taken from  openvino code `make_shared_nv12_blob`